module/VuFind/src/VuFind/RecordDriver/Feature/HierarchyAwareTrait.php

```diff
@@ -194,12 +194,11 @@ public function isCollection()
             case 'All':
                 return isset($this->fields['is_hierarchy_id']);
             case 'Top':
-                return isset($this->fields['is_hierarchy_title'])
+                return $this->getHierarchyTopTitle()
```

Comment:
I don't think you need to check `$this->getHierarchyTopTitle()` as part of the condition at all. Now that you're using a method to retrieve the value, you're guaranteed to get a return value, and if it's unset, that value will be an empty array, which will guarantee that the in_array check below will return false. It's probably okay to simplify this.

Reply:
sure thing!